core/common/src/main/java/alluxio/util/io/PathUtils.java

```diff
@@ -1,7 +1,7 @@
 /*
  * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
  * (the "License"). You may not use this work except in compliance with the License, which is
- * available at www.apache.org/licenses/LICENSE-2.0
+ * available at www.apache.org/licenses/LICENSE-2.0 .
```

Comment:
The build includes a license check which makes sure all of the license statements at the start of each file are exactly equal to a certain string. Could you undo this part of the change?